pkg/git/url_parser_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func TestParseURL(t *testing.T) {
-	spec.Focus(t, "Test Parse Git URL", testParseURL)
+	spec.Focus(t, "Test Parse Git URL", testParseURL) // TODO: should this be .Focus?
```

Comment:
Is this a forgotten `.Focus`?

Reply:
I suspect that it works because its the only thing in the function but it probably should not be a focus